helpdesk/tests/test_api.py

```diff
@@ -15,10 +14,15 @@
     HTTP_403_FORBIDDEN,
 )
 from rest_framework.test import APITestCase
+from _datetime import timedelta
+from helpdesk.lib import convert_value
+from django.utils import timezone
+
+frozen_date_time_str = (datetime.datetime.now() - timedelta(days=100)).isoformat()
```

Comment:
I believe the error from the test comes from the `isoformat` function. It does not add the `Z` at the end.

Reply:
It depends on which platform the test is run. I have tried a few dsifferent options and traced the execution through but not identified why sometomes the folowup_set.due_date is rendered into a string from the FakeDateTime object that freezegun creates.
For now the hack works but perhaps at some point the underlying reason will be identified.

Reply:
Oh yes I see, that's a strange behaviour